### PR TITLE
messages page added

### PR DESCRIPTION
**Pull request by nehanautiyal1991**

### CODE REVIEW

File: src/components/MessageList.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        flexDirection: "row",
+        justifyContent: "flex-start"
+    },
+    bubbleContainer: {
+        // marginLeft: "20px",
+        display: "flex",
+        alignItems: "center",
+        margin: "30px"
+    },
+    talkBubbleLeft: {
```
Comment (bmorwood) [inline src/components/MessageList.js]: `talkBubbleLeft` `talkBubbleright` should be one style like `talkBubble` and then we create two new styles that change the a few specific values

Comment (bmorwood) [reply src/components/MessageList.js]: you can do this with clsx e.g. `<bubble styleName={clsx(classes.textBubble, classes.textBubbleLeft)} /> `

File: src/pages/Meetings.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import RequestMeeting, { meeting_type } from "../components/RequestMeeting";
 import user from "../models/user";
 import RequestMeeting1 from "../components/RequestMeeting1"
 
+
 const Meetings = props => {
   return <RequestMeeting user={user} meeting_type={meeting_type.clientRequestMeeting} />;
   //return <RequestMeeting1 user={user} />;
```
Comment (bmorwood) [inline src/pages/Meetings.js]: remove comment

File: src/components/MessageList.js (hunk 1 above)
Comment (nehanautiyal1991) [reply src/components/MessageList.js]: okay barry

File: src/pages/Meetings.js (hunk 2 above)
Comment (nehanautiyal1991) [reply src/pages/Meetings.js]: sure

**Pull request merged**